src/database.js

```diff
@@ -862,8 +903,8 @@ Database.prototype.runStream = function(query, options) {
     };
   }
 
-  delete reqOpts.toJSON;
-  delete reqOpts.toJSONOptions;
+  delete reqOpts.json;
```

Comment:
Hi, why need `delete reqOpts.json;`? if the query didn't pass `params` or `types` it will return same reference of `query` object, which cause `query.json` also being deleted.